packages/plugins/RedPacket/src/SiteAdaptor/FireflyRedPacketDetailsItem.tsx

```diff
@@ -0,0 +1,245 @@
+  })
+
+  return (
+    <ListItem className={classes.root}>
+      <section className={classes.contentItem}>
+        <Box className={classes.box}>
+          <Box className={classes.content}>
+            <section className={classes.section}>
+              <div className={classes.div}>
+                <div className={classes.fullWidthBox}>
+                  <Typography variant="body1" className={cx(classes.title, classes.message)}>
+                    {rp_msg === '' ?
+                      t.best_wishes()
+                      : rp_msg}
+                  </Typography>
+                </div>
+                <div className={classes.fullWidthBox}>
+                  <Typography variant="body1" className={cx(classes.infoTitle, classes.message)}>
+                    {t.create_time()}
+                  </Typography>
+                  <Typography
+                    variant="body1"
+                    className={cx(classes.info, classes.message, redpacket_id ? '' : classes.invisible)}>
+                    {t.history_duration({ time: dateTimeFormat(new Date(Number(create_time) * 1000)) })}
+                  </Typography>
+                </div>
+                <div className={classes.fullWidthBox}>
+                  <Typography variant="body1" className={cx(classes.infoTitle, classes.message)}>
+                    {t.creator()}
+                  </Typography>
+                  <Typography
+                    variant="body1"
+                    className={cx(classes.info, classes.message)}>
+                    {isAddress(creator) ? formatEthereumAddress(creator, 4) : creator}
+                  </Typography>
+                </div>
+              </div>
+            </section>
+            <section className={classes.footer}>
+              <div style={{ display: 'flex', alignItems: 'center' }}>
+                <Typography variant="body1" className={classes.footerInfo}>
+                  <RedPacketTrans.history_claimed
+                    components={{
+                      span: <span />,
+                    }}
+                    values={{
+                      claimedShares: String(claim_numbers),
+                      shares: String(total_numbers),
+                      amount: formatBalance(total_amounts, token_decimal ?? 18, {
+                        significant: 2,
+                        isPrecise: true,
+                      }),
+                      claimedAmount:
+                        formatBalance(
+                          claim_amounts,
+                          token_decimal,
+                          { significant: 2, isPrecise: true },
+                        ),
+                      symbol: token_symbol,
+                    }}
+                  />
+                </Typography>
+                {token_logo ?
+                  <TokenIcon
+                    className={classes.icon}
+                    address={''}
```

Comment:
It should be a valid EVM address.

Reply:
Data source does not contain token address. Address is not required when we have the token logo url